packages/foam-core/src/utils/hashtags.ts

```diff
@@ -0,0 +1,22 @@
+import { isSome, isNumeric } from './core';
+import matchAll from 'string.prototype.matchall';
```

Comment:
Woah, I didn't know you could do this!

Reply:
unfortunately atm we use node 10, which doesn't have `string.matchAll`, so I added this shim as a dependency (this is an actual dependency, the package name can be misleading though!). 
I don't love it, but updating to node 12 just for this seemed a bit overkill.
Nonetheless I see no problem in doing the update in the future, just wanted to keep it isolated and understand if there are possible side effects :)